<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>603</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>rBackup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="jobs">
       <attribute name="title">
        <string>Jobs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Jobs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="newButton">
              <property name="text">
               <string>New</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="editButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="runButton">
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="enableButton">
              <property name="text">
               <string>Enable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="jobName"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Destination</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="destination"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseDest">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="source"/>
            </item>
            <item>
             <widget class="QPushButton" name="browseSrc">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Options</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="2">
             <widget class="QComboBox" name="backupCompression">
              <item>
               <property name="text">
                <string>No Compression</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>tar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>tar.gz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>tar.bz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>tar.xz</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="deleteWhen">
              <item>
               <property name="text">
                <string>Delete During</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Delete After</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Delete Before</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="transferCompression">
              <property name="text">
               <string>Transfer Compression</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="backupType">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Incremental Backup</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Incremental Backup W/o Delta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Full Backup</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Full Backup W/o Delta</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="generateButton">
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Backup Command</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPlainTextEdit" name="command"/>
          </item>
          <item row="10" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="finish">
              <property name="text">
               <string>Finish</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QCheckBox" name="monday">
              <property name="text">
               <string>Mon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="tuesday">
              <property name="text">
               <string>Tues</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="wednesday">
              <property name="text">
               <string>Wed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="thursday">
              <property name="text">
               <string>Thurs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="friday">
              <property name="text">
               <string>Fri</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="saturday">
              <property name="text">
               <string>Sat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sunday">
              <property name="text">
               <string>Sun</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="recurring">
              <property name="text">
               <string>Recurring</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Date/Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionService_Path"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionService_Path">
   <property name="text">
    <string>Service Path</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
